dashboard/config/scripts/netsim.script

```diff
@@ -14,7 +14,7 @@ level 'NetSim_ManualDNS'
 level 'NetSim_AutomaticDNS'
 
 stage 'Internet Simulator Templates'
-level 'blockly:NetSim:variant1'
-level 'blockly:NetSim:variant2'
-level 'blockly:NetSim:variant3'
+level 'NetSim_Variant1'
+level 'NetSim_Variant2'
+level 'NetSim_Variant3'
```

Comment:
I had to add pass-through `.level` files for these templates, because Laurel and I found a quirk of our level/script system that causes the old and new systems to not play nice together - it builds, but you can get level ID conflicts and the dashboard will think you're on two levels at once.